rele/worker.py

```diff
@@ -48,6 +49,15 @@ def start(self):
                 )
             )
 
+    def run(self, wait=60):
```

Comment:
A `run` that ends after a given number of seconds does not make things much easier. I would instead implement a `run_forever(sleep_interval=2)`, for example.

Reply:
Yea this would be great, maybe something like: 

```

Reply:
By popular demand, I added the `run_forever` method. 👍